### ISSUE TITLE
Error Received When Trying to Train 2 New Users

### ISSUE THREAD

**Issue by pasafier** [nambar12/matlab_netbatch_scheduler#37]
I attempted to train a couple of guys in my group and they both received this error when trying to invoke a pool. We tried several pool sizes, even just 2 workers and always the same error. The first command they entered was this:

nb = generateNetbatchProfile('pdx_dts','/ptm/ipag'); There was no issue with this command.

Then, this command threw the error image below.

pool = parpool(2);

![image](https://user-images.githubusercontent.com/99370541/220714072-f896f340-3ccd-4502-b8d9-a89a05d0768a.png)

**Comment by rsnorris** [comment]
@pasafier please run the following

```
pctconfig('preservejobs',true);
nb = parcluster;
nb.AdditionalProperties.EnableDebug = true;
pool = nb.parpool(2);
nb.getDebugLog(nb.Jobs(end))
```

**Comment by pasafier** [comment]
I had one of them run those commands and the output (a lot!) is below (very bottom). However, what stands out is that there is a license error. I had him add the following line into his cshrc file:

setenv MLM_LICENSE_FILE ***@***.******@***.***>

He then re-ran and got a new error, which is immediately below here:

***@***.***

Here is the log that seems to indicate the license error.


***@***.***

***@***.***


***@***.***
***@***.***
***@***.***
***@***.***

**Comment by rsnorris** [comment]
I'm missing a lot of the debug log, but I suspected it would be related to the license.  Let's chat more about it during our call on Monday, February 27th.

**Comment by pasafier** [comment]
Yes, we can, but I have to send commands to someone else to test since the errors are not with my usage.

**Comment by pasafier** [comment]
Oh that’s interesting and surprising. I’ll have them add those paths asap.
